userguide/tasks.rst

```diff
@@ -811,453 +813,340 @@ Replication Tasks
+   |                           |           |                |                                                                                                                 |
+   |                           |           |                | * *SSH* is supported by most systems. It requires previously created :ref:`SSH Connections`.                    |
+   |                           |           |                | * *SSH+NETCAT* uses SSH to establish a connection to the destination system, then uses                          |
+   |                           |           |                |   `nc(1) <https://www.freebsd.org/cgi/man.cgi?query=nc>`__ to send an unencrypted data stream for higher        |
+   |                           |           |                |   transfer speeds. This is only an option when replicating to a FreeBSD system that has                         |
+   |                           |           |                |   `py-libzfs <https://github.com/freenas/py-libzfs>`__ installed.                                               |
+   |                           |           |                | * *LOCAL* replicates snapshots to another dataset on the same system.                                           |
+   |                           |           |                | * *LEGACY* uses the legacy replication engine from FreeNAS 11.2 and earlier.                                    |
+   |                           |           |                |                                                                                                                 |
+   +---------------------------+-----------+----------------+-----------------------------------------------------------------------------------------------------------------+
+   | SSH Connection            | SSH, NCT, | drop-down menu | Choose the SSH connection to use for the replication. Choose from a list of connections configured in           |
+   |                           | LEG       |                | :ref:`SSH Connections`.                                                                                         |
+   +---------------------------+-----------+----------------+-----------------------------------------------------------------------------------------------------------------+
+   | Netcat Active Side        | NCT       | drop-down menu | System that will open TCP ports for the direct connection.                                                      |
+   +---------------------------+-----------+----------------+-----------------------------------------------------------------------------------------------------------------+
+   | Netcat Active Side Listen | NCT       | string         | Network IP address that the :guilabel:`Active Side` binds to. Defaults to :literal:`0.0.0.0`.                   |
```

Comment:
This whole "active side" part is not clearly defined.

Reply:
yep